codewof/programming/content/en/vertical-line/question.md

```diff
@@ -0,0 +1,5 @@
+#Vertical Line
+
+Write a function `is_vertical(point1, point2)` that takes two points represented by tuples of the form (x, y).
+
+It should return True if the points lie on a vertical line, or False otherwise.
```

Comment:
Modify to be similar to the suggestion for the horizontal line question.

Reply:
Done